packages/cadl-ranch-specs/http/client/structure/default/client.tsp

```diff
@@ -6,7 +6,7 @@ import "@azure-tools/typespec-client-generator-core";
   """
   This is to show that if we don't do any customization. The client side should be able to call the api like
   ```ts
-  const client = new MultiClient("default");
+  const client = new ServiceClient("default");
```

Comment:
should we update the call for seven, eight and nine ? I might get a little confused, what's the default behavior for hierarchy client is ?  do we treat namespaces as hierarchies or not ?

Reply:
if yes, should it be client.foo.three() and client.baz.foo.seven() ?

Reply:
yes. it should be `client.baz.foo.seven()`